src/components/CustomCollection/EditableCollectionTitle.tsx

```diff
@@ -58,9 +59,17 @@ export const EditableCollectionTitle = ({
             required
             maxLength={200}
             className={styles.collectionTitle}
-            defaultValue={text}
+            value={currentText}
             placeholder={`Name this collection`}
             type="text"
+            onKeyDown={(e) => {
+              if (e.key === ' ') {
+                e.preventDefault()
+                const userInput = currentText + ' '
+                setCurrentText(userInput)
+              }
+            }}
+            onChange={(e) => setCurrentText(e.target.value)}
```

Comment:
The reason for adding this, is that when creating a new collection in My Space the user has to type a title for the collection, and if they hit 'space' it fires the `KeyboardSensor` for `dnd-kit`. This causes there to be no space added to the input field, and if the user hits the arrow keys the collection moves around. This is a hacky workaround, and I'm not sure why, but I don't like that I had to do this. It works as expected, but just feels strange.